src/components/learner-credit-management/cards/AssignmentModalSummaryLearnerList.jsx

```diff
@@ -30,13 +30,14 @@ const AssignmentModalSummaryLearnerList = ({
         {displayedLearnerEmails.map((emailAddress) => (
           <li key={uuidv4()} className="small">
             <div className="d-flex justify-content-between">
-              <div>
+              <div style={{ maxWidth: '85%' }}>
```

Comment:
[inform] Needed to set a `maxWidth` for the truncation of looooong email addresses to get the overflow ellipsis. `85%` was deemed reasonable.